notebook_index.yml

```diff
@@ -0,0 +1,164 @@
+meta:
+  author: Dan Gunter
+front_matter:
+  -
+    title: Introduction
+    text: >-2
+      The [IDAES](https://www.idaes.org) modeling platform ships with a number of examples which can be run on
+      the user's own computer.
+      This page provides links to these examples and provides some guidance in the order in which to try them.
+
+
+      The IDAES examples are contained in Jupyter Notebooks. In order to view and
+      use this content, you need to open the files with the Jupyter notebook executable
+      (which may be configured on your system as the default application for files of this
+      type). To get started with Jupyter, please
+      see the [Jupyter website](https://jupyter.org) or jump directly to the
+      [official Jupyter Notebook documentation pages](https://jupyter-notebook.readthedocs.io/en/stable/).
+
+
+      In addition to viewing and running the examples interactively on your own computer, you can see fully rendered,
+      static versions of the examples in the online
+      [examples documentation](https://IDAES.github.io/examples-pse/) pages. For reference documentation
+      on the IDAES core modeling framework, please see the online
+      [IDAES-PSE documentation](https://idaes-pse.readthedocs.io/en/stable/index.html).
+  -
+    title: Usage
+    text: >-2
+      The example notebooks contained in this folder are divided into two sub-types, each with their own folder:
+        * `Tutorials`: Notebooks that are written as tutorials complete with guided exercises
+        * `Examples`: Notebooks that do not have tutorial content.
```

Comment:
I like this distinction. Currently, based on this definition, only the workshop modules will be classified as Tutorials. All the other notebooks are examples that do not have any exercise cells.